test/index.test.ts

```diff
@@ -0,0 +1,197 @@
+import { install } from "lolex";
+import nock from "nock";
+import { stub } from "simple-mock";
```

Comment:
I think `jest` has it’s own stubbing, maybe we use that instead of simple-mock?
https://jestjs.io/docs/en/mock-function-api.html

We can also do it in a follow up PR